news/2022-02-02-new-featured-artist-bill-wurtz.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "new featured artist: bill wurtz"
+date: 2022-02-02 09:00:00 +0000
+---
+
+hello.
+
+![](https://assets.ppy.sh/artists/236/header.jpg)
+
+music by **bill wurtz.**
+
+**21** songs.
+
+samples:
+
+### La de da de da de da de day oh
+
+[circle beatmap](https://osu.ppy.sh/beatmapsets/725234) by [mindmaster107](https://osu.ppy.sh/users/8819496)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### outside
+
+[piano beatmap](https://osu.ppy.sh/beatmapsets/735580) by [Nivrad00](https://osu.ppy.sh/users/1984634)
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### and the day goes on
+
+[nothing beatmap](LINK)
```

Comment:
would greatly prefer `beatmap template 1` and `beatmap template 2` over `nothing beatmap 1` and `nothing beatmap 2` tbh, "nothing beatmap" just... doesn't feel quite right :(

Reply:
i think it fits the style better as is (going from circles to piano to nothing)